doc/Journal.tex

```diff
@@ -220,143 +220,113 @@ \section{Background}
-scatter in the air outside of the cone, though the mask, and into the detector is negligible.  
+scatter in the air outside of the cone, though the mask, and into the detector will have negligible contributions to the simulated DRCs.   
 In addition, a 0.095 inch air gap between the mask and aluminum sleeve constrains the mask geometry from impinging on the sleeve and provides a space for grease to be applied between the moving parts.
-Also, note that due to manufacturing constraints each mask angle has a non-zero thickness.
-
-\section{Design Assumptions}
-Monte Carlo N-Particle (MCNP) is one method used to simulate energy spectrum.  
-Logan et al.\cite{Logan2017} showed statistical agreement between experimental and simulated DRCs.  
-Thus, this work will use simulated DRCs to evaluate the RSM performance.  
-Instead of the full energy peak (FEP), the DRC is formed by adding all counts above 200keV.  
-This value was chosen as Logan et al. noted discrepancies due to scatter in the enclosing environment for counts below approximately 200keV.
-
-Herein, it is assumed that the geometry is related to the DRC.  
-Qualitative studies of this correlation showed that in general, this assumption is valid with two qualifications.  
-First, a discontinuous geometry results in a continuous DRC.  
-To understand the next qualification, define a cell to be a rectangular prism with a given thickness extending from the centroid of the detector (outside of the Aluminum sleeve) in a given direction.
+Finally, due to manufacturing constraints, each mask angle must have a non-zero thickness.
+
+\subsection{Design Assumptions and Limitations}
+% This paragraph in general has me a bit confused.  I added the "detector-mask-source" geometry as I think this is what was being referred to.   
+It is assumed that the detector-mask-source geometry is related to the DRC and that geometry can be reconstructed using the DRC.
+Qualitative studies of this correlation showed that, in general, this assumption is valid with two qualifications.  
+%I am not sure how the first qualificiation fits into the preceeding statements.  Is the point that in contrast to Fitzgerald's design assumptions?
+First, a discontinuous geometry results in a continuous DRC. 
+% Here, I'd start with stating the qualification, then explaining it. 
+To understand the next qualification, define a cell to be a rectangular prism with a given thickness extending from the centroid of the detector (outside of the aluminum sleeve) in a given direction.
 Since the cells do not have impenetrable walls, particles from one source position enter cells pointed at other positions.  
+% Is it the increase in scatters that leads to a higher number of counts or the reduction in photon energy that leads to a higher interaction probability?  As written, I read it as scatters are preferentially towards the detector, or that there is some multiplication happening, both of which aren't true.
```

Comment:
In contrast to an attenuation mask, (as I understand Fitzgerald's design) one of a rotation scatter mask's beneficial charsince particles which would not make it into the detector previously, but pass by to one side (or are blocked by the mask material) can now be scattered into the detector increasing the number of events in the detector.  This does not mean that they are useful for direction identification.

Reply:
After thinking about this a bit, I see what you are saying.  Since CS is preferentially forward you still capture a lot of the radiation in the detector's solid angle, but the mask can scatter some that you would otherwise have no chance of.